.stylelintrc.json

```diff
@@ -0,0 +1,24 @@
+{
+    "extends": ["stylelint-config-standard-scss"],
+    "rules": {
+      "at-rule-no-unknown": null,
+      "at-rule-empty-line-before": ["always", { "except": ["after-same-name", "inside-block", "blockless-after-same-name-blockless"], "ignoreAtRules": ["function", "mixin"] }],
+      "block-closing-brace-newline-after": ["always", { "ignoreAtRules": ["if", "else"] }],
+      "block-opening-brace-space-before": "always",
+      "block-opening-brace-newline-after": "always",
+      "block-closing-brace-newline-before": "always",
+      "block-closing-brace-empty-line-before": "never",
+      "declaration-block-trailing-semicolon": "always",
+      "declaration-empty-line-before": "never",
+      "declaration-block-semicolon-newline-after": "always-multi-line",
+      "font-family-name-quotes": "always-where-required",
+      "indentation": 2,
+      "max-line-length": null,
+      "no-descending-specificity": null,
```

Comment:
Once/if this PR gets approved, we should create a separate ticket to re-enable this rule. I disabled it because it was flagging several selectors in `www/styles/docs.scss` that violated the rule. I didn't want to introduce any breaking changes there, so I disabled it for safe measure.

Reply:
If only we could comment JSON... will make an issue for this 🙃